tabpy-server/tabpy_server/psws/callbacks.py

```diff
@@ -36,7 +35,6 @@ def wait_for_endpoint_loaded(python_service, object_uri):
         sleep(0.1)
 
 
-@gen.coroutine
```

Comment:
Items in this module didn't appear to be doing anything asynchronous so simply removed the decorator and did not add async in contrast to other methods